frontend/packages/dev-console/integration-tests/features/addFlow/create-from-git.feature

```diff
@@ -203,3 +203,13 @@ Feature: Create Application from git form
                   | git_url                                |
                   | https://github.com/sclorg/httpd-ex.git |
                   | https://github.com/sclorg/nginx-ex.git |
+
+        @regression @manual
```

Comment:
Btw, it would be still cool if this is not a manual (manually clicked through the UI) test and the statements below could run via cypress.

Reply:
Yup, removed manual. Since it is only a single API call, I don't think it would affect the rate limit issue that much.